### ISSUE TITLE
First time setup - signing in to connect with Youtube Music - Couldn't sign you in - Browser or app may not be secure

### ISSUE THREAD

**Issue by bravoland** [jamesbrindle/YTMusicUploader#30]
is there anything special i need to setup?
i click the connect to Youtube Music
I type in my UN/email and click next and then it shows
borwers or app not secure with no options

![image](https://user-images.githubusercontent.com/35783858/127348097-a5af10f4-43d6-4eef-aa34-709860977ef5.png)

**Comment by illMadeKnight** [comment]
+1 I have the same issue.

**Comment by jamesbrindle** [comment]
Hey,

I had to remove the embedded browser for the 'Connect to YTM' as google recently prevented sign-in via embedded browsers (https://portswigger.net/daily-swig/google-to-block-sign-in-attempts-from-embedded-browsers). Therefore, as a workaround, I've opted to simply allow users to paste their authentication cookie value into a text box to get the app working. 

A video tutorial link is found within the 'Connect To YTM' Form.

New version here: https://github.com/jamesbrindle/YTMusicUploader/releases/tag/v1.8.0
